stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/extensions/ClientExtensions.kt

```diff
@@ -137,6 +141,24 @@ public fun ChatError.isPermanent(): Boolean {
     return isPermanent
 }
 
+public fun ChatClient.domain(): ChatDomain {
```

Comment:
Extensions are neat, but let's consider what using these will look like for Java clients